commands/Tools/contact.js

```diff
@@ -0,0 +1,71 @@
+exports.init = async (client) => {
+  const schema = client.settings.guilds.schema;
+  if (!schema.bugChannelId) {
+    await client.settings.guilds.add("bugChannelId", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+  if (!schema.ideaChannelId) {
+    await client.settings.guilds.add("ideaChannelId", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+  if (!schema.messageChannelId) {
+    await client.settings.guilds.add("titleURL", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+  if (!schema.reportChannelId) {
+    await client.settings.guilds.add("reportChannelId", {
+      type: "String",
+      default: null,
+      array: false,
+      sql: "TEXT NOT NULL DEFAULT ','",
+    });
+  }
+};
+
+exports.run = async (client, msg, [type, ...content]) => {
+  // Declares all necessary variables to use for the command.
+  const { bugChannelId, ideaChannelId, messageChannelId, reportChannelId } = msg.guild.settings;
```

Comment:
If this is going to be a piece, please do not use a config.js/json but a plain channel id.

Also, I would suggest this:
```javascript
exports.channel = null;

exports.init = (client) => {
  this.channel = client.channels.get("CHANNEL_ID");
}
```

And then in the code, you just do `this.channel.send`. It's perfectly fine as Channel will be a [pointer](https://en.wikipedia.org/wiki/Pointer_(computer_programming)) and won't affect memory usage negatively, in fact, it'll perform much better than re-getting this once and again.